src/main/java/com/algomart/kibouregistry/entity/Participants.java

```diff
@@ -7,6 +7,10 @@
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
 import java.util.List;
```

Comment:
You should not use Javax because it has been migrated to jakarta. Change your validations annotations to use Jakarta validation.

Reply:
The dependency is Jakarta but the annotation is Javax. There is no annotation for Jakarta, it is Javax

Reply:
Are you kidding me with this reply, sir? No annotation for Jakarta you say? Please always ask questions when asked to do something and you are not clear. Check the screenshot below and let me know if there is no annotation for Jakarta. 
![Screenshot 2024-03-22 at 16 55 08](https://github.com/algo-mart/kibou-registry/assets/100934970/63e28992-42b8-4906-aa87-f64b71e1b03a)

Reply:
Oh! Sorry. I thought you are making reference to @valid in controller

Reply:
Well right above there...It's written Participants.java which means it is an entity class, not a controller class. And even at that @Valid annotation is present in Jakarta. You are mixing things up because you added Javax dependency to your pom. You should remove it from your Pom file